index.bs

```diff
@@ -2421,6 +2415,106 @@ This is because the first 37 bytes of the signed data in a FIDO U2F authenticati
 the same procedure as other [=assertion signatures=] generated by the [=authenticatorMakeCredential=] operation.
 
 
+## Authenticator taxonomy ## {#sctn-authenticator-taxonomy}
+
+Different kinds of [=authenticators=] enable different use cases along two independent dimensions: the [=authentication ceremony
+structure=] and the [=authenticator=]'s [=attachment modality=]. The [figure below](#fig-authData) illustrates how the use cases
+relate to different kinds of [=authenticators=].
```

Comment:
Thanks! I'll take that suggestion as is.

---

wrt https://github.com/w3c/webauthn/pull/842#discussion_r195583403 (can't comment on it for some reason): Yes. Perhaps it would be better with just a big table of YESs and NOs instead of the figure? Something like:

 UV? | RK? | 2nd factor | Passwordless | Username-less 
:---:|:---:|:----------:|:------------:|:-------------:
 No  | No  |    Yes     |      No      |      No
 No  | Yes |    Yes     |      No      |      Yes
 Yes | No  |    Yes     |      Yes     |      No
 Yes | Yes |    Yes     |      Yes     |      Yes